nimble/host/audio/include/audio/ble_audio_broadcast_sink.h

```diff
@@ -0,0 +1,267 @@
+ * under the License.
+ */
+
+#ifndef H_BLE_AUDIO_BROADCAST_SINK_
+#define H_BLE_AUDIO_BROADCAST_SINK_
+
+/**
+ * @file ble_audio_broadcast_sink.h
+ *
+ * @brief Bluetooth LE Audio BAP Broadcast Sink API
+ *
+ * @defgroup ble_audio_broadcast_sink Bluetooth LE Audio BAP Broadcast Sink
+ * @ingroup bt_host
+ * @{
+ *
+ */
+
+#include <stdint.h>
+#include "host/ble_gap.h"
+#include "host/ble_iso.h"
+#include "audio/ble_audio.h"
+#include "audio/ble_audio_scan_delegator.h"
+#include "nimble/ble.h"
+
+enum ble_audio_broadcast_sink_action_type {
+    /** Broadcast Sink Action Type: PA sync */
+    BLE_AUDIO_BROADCAST_SINK_ACTION_PA_SYNC,
+
+    /** Broadcast Sink Action Type: BIG sync */
+    BLE_AUDIO_BROADCAST_SINK_ACTION_BIG_SYNC,
+
+    /** Broadcast Sink Action Type: BIS sync */
+    BLE_AUDIO_BROADCAST_SINK_ACTION_BIS_SYNC,
+
+    /** Broadcast Sink Action Type: Start discovery (scan) */
+    BLE_AUDIO_BROADCAST_SINK_ACTION_DISC_START,
+
+    /** Broadcast Sink Action Type: Start discovery (scan) */
+    BLE_AUDIO_BROADCAST_SINK_ACTION_DISC_STOP,
+};
+
+struct ble_audio_broadcast_sink_action {
+    /**
+     * Indicates the type of action that is requested.
+     */
+    enum ble_audio_broadcast_sink_action_type type;
+
+    /**
+     * A discriminated union containing additional details concerning the action.
+     * The 'type' field indicates which member of the union is valid.
+     */
+    union {
+        /**
+         * Represents PA Sync parameters request.
+         *
+         * The action triggered on locally or remotely initiated PA synchronization request.
+         * The application should initialize the `out_parameters`, or abort the process.
+         *
+         * Valid for the following action types:
+         *     o BLE_AUDIO_BROADCAST_SINK_ACTION_PA_SYNC
+         *
+         * Return:
+         *     o 0 on success;
+         *     o A non-zero value to abort.
+         */
+        struct {
+            /** Pointer to Periodic Sync parameters to initialize. */
+            struct ble_gap_periodic_sync_params *out_params;
+        } pa_sync;
+
+        /**
+         * Represents BIG Sync request.
+         *
+         * The action triggered on locally or remotely initiated BIG synchronization request.
+         * The application should provide the `out_mse` and `out_sync_timeout`,
+         * or reject the request.
+         *
+         * Valid for the following action types:
+         *     o BLE_AUDIO_BROADCAST_SINK_ACTION_BIG_SYNC
+         *
+         * Return:
+         *     o 0 on success;
+         *     o A non-zero value to abort.
+         */
+        struct {
+            /** Source ID. */
+            uint8_t source_id;
+
+            /** ISO Interval. */
+            uint16_t iso_interval;
+
+            /** Presentation delay. */
+            uint32_t presentation_delay;
+
+            /** Pointer to Maximum Subevents value to initialize. */
+            uint8_t *out_mse;
```

Comment:
We should add NSE and BN here so application has any means to choose out_MSE.

Also we should set default to 0x00 which means controller can choose up to NSE

Reply:
done